### PR TITLE
[TE] Self-Serve Onboarding 07: enhancements v1

### PR DESCRIPTION
**Pull request by newsummit**
- Fix change rate value in anomaly table (and color)
- Getting graph to react to table dimension selection

In commits 2 and 3:
- Better documenting "dense" metricDataUrl builder helper
- Removing "projected" values in Alert Page
- CSS adjustments for text overflow cases
- Using colors defined in app

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/styles/shared/_styles.scss [hunk 1]
```diff
@@ -541,18 +541,30 @@ body {
...
+  &__list-item {
+    font-size: 12px;
+    margin-left: -2px;
+  }
+
+  &__value-label {
+    font-size: 13px;
+
+    &--up {
+      color: green;
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/styles/shared/_styles.scss]: To keep a consistent UI, let's pick colors from here https://github.com/linkedin/pinot/blob/master/thirdeye/thirdeye-frontend/app/styles/wrapper/_color.scss

Comment (newsummit) [reply thirdeye/thirdeye-frontend/app/styles/shared/_styles.scss]: will do.

File: thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/template.hbs [hunk 2]
```diff
@@ -80,6 +69,17 @@
       {{/power-select}}
     </div>
 
+    <div class="col-xs-6">
```
Comment (newsummit) [inline thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/template.hbs]: Switching "filters" and "dimensions" fields (dimension to the left)

Comment (justYves) [reply thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/template.hbs]: Did we check with design that this change is fine? (not sure if Selene had the previous order for a specific reason)

File: thirdeye/thirdeye-frontend/app/utils/manage-alert-utils.js [hunk 3]
```diff
@@ -305,7 +308,7 @@ export function buildAnomalyStats(alertEvalMetrics, mode, severity = '30', isPer
     let newData = alertEvalMetrics.projected[stat.key];
     let isPercentageMetric = stat.units === '%';
     let isTotal = stat.key === 'totalAlerts';
-    stat.showProjected = mode === 'explore';
+    stat.showProjected = false;
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/utils/manage-alert-utils.js]: Let's add a comment that this is temporary + JIRA Ticket tracking the status of the fix

File: thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/template.hbs (hunk 2 above)
Comment (newsummit) [reply thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/template.hbs]: will check

File: thirdeye/thirdeye-frontend/app/utils/manage-alert-utils.js (hunk 3 above)
Comment (newsummit) [reply thirdeye/thirdeye-frontend/app/utils/manage-alert-utils.js]: ok

**Pull request merged**